section3/reflection.md

```diff
@@ -2,12 +2,23 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
+Once again it felt daunting, but I did feel myself gain more momentum through out this section. It was motivating.
+
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
 
+Arrays are used everywhere. One example is the app called Plant Id. You can type in a plant and it will display a list of characteristics associated with that plant.
+
 3. In your own words, explain what an Array method is.
 
+When we want to change the information in any given array, we use array methods. Each method performs different tasks depending on the desired result.
+
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
 
+We use loops when we want our program to perform an action a specific number of times. We would use a loop in conjunction with an array when we want to perform an action on each item in that array. We wouldn't use a loop in conjunction with an array when we don't need to access all of items in the array.
+
 5. If you had to teach a 5 year old about loops, how would you explain it?
 
+I would ask the child to think about a task where they have to do the same thing multiple times. For example, lets say the child has to clean up the table after supper. You gather a plate. Then you bring that plate to the sink. Next, you wash the plate. Last, you dry it. But, what if your family is huge and you have to do this for several plates? Perhaps this means that you have to do the same task several times. Computers need different instructions to execute their jobs. So if you wanted to make a robot do your dishes, you would have to tell the robot how to do this job in robot language. So you would need to write in each action, or each code, one at a time. To save time, we use loops so we don't have to type in the action every time.
+
 6. What questions do you still have about Arrays and loops?
+I would love further clarification on when to use loops without arrays. 
```

Comment:
If you want to do something many times, but that "thing" isn't dependent on data in an array.

In real world applications, we might use it for a countdown "how many days until the wedding?" or something like that. More often, we do use loops in conjunction with arrays. Fundamentally, it's important you know that a for loop is not inherently tied to an Array, so that's really why we force you to write them some without Arrays, to help illustrate that.